TLM/TLM/Manager/Impl/VehicleBehaviorManager.cs

```diff
@@ -710,7 +678,11 @@ public class VehicleBehaviorManager : AbstractCustomManager, IVehicleBehaviorMan
 			return maxSpeed;
 		}
 
-		public uint GetVehicleRand(ushort vehicleId) {
+		public uint GetStaticVehicleRand(ushort vehicleId) {
+			return vehicleId % 100u;
+		}
+
```

Comment:
This should not be here. It's part of another bugfix which is not related to the turn-on-red feature